### PR TITLE
#3: Added configuration for Travis CI.

### PR DESCRIPTION
**Pull request by redcatbear**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # junit5-system-extensions (J5SE)
 
-This project provides a set of JUnit 5 extension that allow testing behavior related to functions related to `java.lang.System` (e.g. asserting exit status codes). 
+This project provides a set of JUnit 5 extension that allow testing behavior related to functions related to `java.lang.System` (e.g. asserting exit status codes).
+
+[![Build Status](https://travis-ci.com/itsallcode/junit5-system-extensions.svg?branch=master)](https://travis-ci.com/itsallcode/junit5-system-extensions)
```
Comment (kaklakariada) [inline README.md:5]: Branch should be develop?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+cache:
+  directories:
+  - $HOME/.m2
+  - $HOME/.sonar/cache
+
+before_script:
+  - pip install --user codecov
+
+before_install:
+  - echo -n | openssl s_client -connect scan.coverity.com:443 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' | sudo tee -a /etc/ssl/certs/ca-
```
Comment (kaklakariada) [inline .travis.yml]: Not necessary, we don't use coverity here.

**Review by kaklakariada — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (redcatbear) [reply README.md:5]: I think we should change our strategy and make "master" the default branch in our repositories again. For us that is inconvenient, but it is what users expect.

File: .travis.yml (hunk 2 above)
Comment (redcatbear) [reply .travis.yml]: Removed

**Comment by redcatbear** [comment]
@kaklakariada, please review again.

**Pull request merged**